packages/e2e-tests/specs/performance.test.js

```diff
@@ -0,0 +1,69 @@
+/**
+ * External dependencies
+ */
+
+import { writeFileSync } from 'fs';
+
+/**
+ * WordPress dependencies
+ */
+import {
+	createNewPost,
+	saveDraft,
+	insertBlock,
+} from '@wordpress/e2e-test-utils';
+
+describe( 'Performance', async () => {
+	it( '1000 paragraphs', async () => {
```

Comment:
Do we want it to be executed every time we run e2e tests?

Reply:
No, let's exclude.